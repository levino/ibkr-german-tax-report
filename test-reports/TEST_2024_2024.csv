Statement,Header,Field Name,Field Value
Statement,Data,Title,Activity Statement
Statement,Data,Period,"January 1, 2024 - December 31, 2024"
Statement,Data,WhenGenerated,"2024-12-31, 15:30:00 EST"
Account Information,Header,Field Name,Field Value
Account Information,Data,Account,T1234567
Dividends,Header,Currency,Date,Description,Amount
Dividends,Data,EUR,2024-06-15,XYZ(US98765XYZ43) Cash Dividend EUR 2.50 per Share (Ordinary Dividend),100
Dividends,Data,EUR,2024-07-20,ABC(DE000ABC1234) Cash Dividend EUR 1.25 per Share (Ordinary Dividend),50
Dividends,Data,Total,,,150
Withholding Tax,Header,Currency,Date,Description,Amount,Code
Withholding Tax,Data,EUR,2024-06-15,XYZ(US98765XYZ43) Cash Dividend EUR 2.50 per Share - DE Tax,-15,
Withholding Tax,Data,EUR,2024-07-20,ABC(DE000ABC1234) Cash Dividend EUR 1.25 per Share - DE Tax,-7.5,
Withholding Tax,Data,Total,,,-22.5,