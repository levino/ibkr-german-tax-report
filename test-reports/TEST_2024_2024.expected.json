{
  "7": "150",
  "19": "0",
  "37": "21.33",
  "38": "1.17",
  "41": "0"
}